src/components/AddOn/core/AddOnStore/AddOnStore.tsx

```diff
@@ -266,7 +260,9 @@ function addOnStore(): JSX.Element {
                 </Tab>
                 <Tab eventKey="installed" title={t('install')}>
                   {data.getPlugins
-                    .filter((plugin: any) => plugin.pluginInstallStatus == true)
+                    .filter(
+                      (plugin: any) => !plugin.uninstalledOrgs.includes(orgId),
+                    )
```

Comment:
@Anvita0305 Can you please explain why this is changed? (cc @SiddheshKukade is this a required change?)

Reply:
Initially, plugin.pluginInstallStatus was being filtered if true. But in the latest schema there is no such field. 
So instead, I am checking if the current orgId is not in uninstalledOrgs list of the plugins.